### DISCUSSION TITLE
Current Port for SLSK Server?

### DISCUSSION THREAD

**Discussion by DeathStalker77** [nicotine-plus/nicotine-plus#3019] [Q&A] [answered]
Greetings all!
The last version of N+ I had to change my server connection port to 2242.  I just installed the current build, and now it won't connect on that port.  What Port (or Ports) should I be using?

I found one page that referenced 2234 - 2239, but I've tried several of them and it's still not connecting.

Thanks!

--- DS

**Comment by mathiascode** [comment, marked as answer]
The default server address is `server.slsknet.org:2242`.

2234 - 2239 are not server ports, it was the old listening port range (now it's just 2234).

**Reply by mathiascode** [reply]
Looks like the server is currently down: https://groups.google.com/g/soulseek-discussion/c/ZGepId6Sdqk

**Comment by DeathStalker77** [comment]
Ok, that's what I was wondering.  Ok, sorry for the terminology - I meant Listening Ports.  I know I have used 2242 (& 2216, I think).

--- DS
